template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ConsoleApp-CSharp/Company.ConsoleApplication1.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework Condition="'$(target-framework-override)' == ''">net6.0</TargetFramework>
+    <TargetFramework Condition="'$(target-framework-override)' != ''">target-framework-override</TargetFramework>
+    <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">Company.ConsoleApplication1</RootNamespace>
+    <LangVersion Condition="'$(langVersion)' != ''">$(ProjectLanguageVersion)</LangVersion>
+    <Nullable Condition="'$(nullable)' == 'true'">enable</Nullable>
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
```

Comment:
check for overflow is enabled by default.
nullable are enabled by default, and controlled via `--nullable`.